packages/tests/tests/dns-peer-discovery.spec.ts

```diff
@@ -86,6 +86,8 @@ describe("DNS Node Discovery [live data]", function () {
   });
 
   it(`should retrieve ${maxQuantity} multiaddrs for test.waku.nodes.status.im`, async function () {
+    if (process.env.CI) this.skip();
```

Comment:
Did you add it to avoid running on old version of Node, @danisharora099 ?

Reply:
This is to avoid running tests against an external environment (e.g. test fleet) in the CI because the CI is then prone to breaking over a change that we cannot control with code.